primedev/scripts/client/scriptmodmenu.cpp

```diff
@@ -151,8 +122,7 @@ ADD_SQFUNC("bool", NSIsModRequiredOnClient, "string modName", "", ScriptContext:
 	return SQRESULT_NULL;
 }
 
-ADD_SQFUNC(
-	"array<string>", NSGetModConvarsByModName, "string modName", "", ScriptContext::SERVER | ScriptContext::CLIENT | ScriptContext::UI)
+ADD_SQFUNC("array<string>", NSGetModVersions, "string modName", "", ScriptContext::SERVER | ScriptContext::CLIENT | ScriptContext::UI)
```

Comment:
shouldn't this be in modinfo as well when you can get modinfo for a singular mod as well. Then the usage in main menu should be easy as well